### PR TITLE
add Integrated Cell to morphological phenotypes

### PR DESCRIPTION
**Pull request by alxndrkalinin**
Added #289 / #372 as an early example of promising DL approach to cell shape modeling

### CODE REVIEW

File: sections/04_study.md [hunk 1]
```diff
@@ -518,6 +518,15 @@ extract more subtle features through its hidden nodes. Already, there is
 evidence deep learning can surpass the efficacy of classical methods
 [@doi:10.1101/081364], even using generic deep convolutional networks trained on
 natural images [@doi:10.1101/085118], known as transfer learning.
+Recent work by Johnson et al. [@tag:Johnson2017_integ_cell] demonstrated how
+the use of a conditional adversarial autoencoder allows for a probabilistic
+interpretation of cell and nuclear morphology and structure localization from
+fluorescence images. Proposed model is able to generalize well to a wide range
```
Comment (agitter) [inline sections/04_study.md]: "The proposed model"

File: sections/04_study.md [hunk 2]
```diff
@@ -518,6 +518,15 @@ extract more subtle features through its hidden nodes. Already, there is
...
+of subcellular localizations. Generative nature of the model allows to produce
```
Comment (agitter) [inline sections/04_study.md]: "The generative" ... "allows it to"

File: sections/04_study.md [hunk 3]
```diff
@@ -518,6 +518,15 @@ extract more subtle features through its hidden nodes. Already, there is
...
+high quality synthetic images predicting localization of subcellular structures
+by directly modeling the localization of fluorescent labels. Notably, this
+approach reduces the modeling time by omitting subcellular structure
```
Comment (agitter) [inline sections/04_study.md]: "omitting the subcellular"

**Comment by alxndrkalinin** [comment]
@agitter thanks, fixed

**Comment by agitter** [comment]
Thanks, it's ready to merge.  I'll leave it open a little longer in case anyone else has comments.

**Pull request merged**